src/index.js

```diff
@@ -1,27 +1,34 @@
+// @flow
+
 /**
  * Multihash implementation in JavaScript.
  *
  * @module multihash
  */
-'use strict'
-
-const bs58 = require('bs58')
-
-const cs = require('./constants')
-
-exports.names = cs.names
-exports.codes = cs.codes
-exports.defaultLengths = cs.defaultLengths
-
-const varint = require('varint')
+import bs58 from 'bs58'
```

Comment:
While now working myself on typing `multibase` I came to ask: Don't you have to include type definitions for this? Or are fine with having all our externals default to any?